lib/telegraph/client.rb

```diff
@@ -0,0 +1,67 @@
+module Telegraph
+  class Client
+    include Telegraph::Connection
+
+    attr_reader :token
+
+    def initialize(token)
+      @token = token
+    end
+
+    def create_account(attrs = {})
+      acc_info = get('createAccount', attrs)
+      @account = Account.new(acc_info)
+    end
+
+    def account
+      @account ||= get_account_info
+    end
+
+    def get_account_info(*fields)
+      # Gets all available account information if 'fields' array not given.
+      available_fields = %w(short_name author_name author_url auth_url page_count)
+      fields = available_fields if fields.empty?
+      response = get('getAccountInfo', fields: fields.to_s)
```

Comment:
I'd not rely on `to_s` here, `to_json` is more robust. In the old/new Ruby versions, `Array#to_s` may suddenly change its formatting. And probably it should be done inside `get`